### ISSUE TITLE
Could you tell me which dataset are the .feat files generated from

### ISSUE THREAD

**Issue by pie-mi** [yuweiwan/ASR-SG-HMM-GMM#1]
'cause the .feat files are up to more than 240000 lines,  I guess it wouldn't be a small dataset. 
btw, why the test set is as big as training? Normally test set is much smaller than the training set, isn't it?
